e2e/extensions.e2e.spec.ts

```diff
@@ -202,7 +203,7 @@ test.describe.serial('Extensions', () => {
       test('capturing output', async() => {
         const script = `
           ddClient.extension.host.cli.exec("${ wrapperName }", [
-            "${ process.execPath }", "-e", "console.log(1 + 1)"
+            "${ execPath }", "-e", "console.log(1 + 1)"
           ]).then(({cmd, killed, signal, code, stdout, stderr}) => ({
```

Comment:
Alternatively:
```suggestion
          ddClient.extension.host.cli.exec("${ wrapperName }", ${
            JSON.stringify([process.execPath, '-e', 'console.log(1 + 1)'])
          }).then(({cmd, killed, signal, code, stdout, stderr}) => ({
```

What you have is fine, and this was more of an intellectual exercise.

Reply:
That looks more complex